release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -2457,6 +2438,38 @@ spec:
 
 * The pipeline metrics API does not support the required `pipelinerun/taskrun` histogram values from {rh-openstack} 1.6 and beyond. Consequently, the *Metrics* tab in the *Pipeline* -> *Details* page is removed instead of displaying incorrect values. There is currently no workaround for this issue. (link: https://bugzilla.redhat.com/show_bug.cgi?id=2074767[*BZ#2074767*])
 
+* In {product-title} {product-version}, pod security admission runs globally with privileged enforcement and restricted audit logging and API warnings. Versions of the `opm` binary released earlier than {product-title} {product-version}, do not work with the restricted profile setting. As a result, an error displays if you install a catalog source that was built with an `opm` binary released earlier than {product-title} {product-version}.
```

Comment:
> * In {product-title} {product-version}, pod security admission runs globally with privileged enforcement and restricted audit logging and API warnings. 

@michaelryanpeter @anik120 @jianzhangbjz   ^ the above text has no relation to the issue ^. Btw, all are still as "Previlged" and that will only change on 4.12. From 4.12, Openshift system namespaces (whch are these ones [here](https://github.com/openshift/openshift-docs/pull/48409/files#diff-6b079749a009ff430faa75c3cdece0921ba0e269d1dcd32d6ec0939c7ff99059R9-R23)) will be enforced to restricted. 

Following my suggestion

> * `opm` binary released earlier than {product-title} {product-version}, do not work with the restricted profile setting. As a result, an error displays if you install a catalog source that was built with an `opm` binary released earlier than {product-title} {product-version} will be raised when the namespace is enforced to be restricted and the pod security admission label synchronization is disabled. Note that Openshift system namespaces do not allow you to enable the pod security admission label synchronization.  (We need a link here to the SCC docs, see https://github.com/openshift/openshift-docs/pull/48409/files#diff-6b079749a009ff430faa75c3cdece0921ba0e269d1dcd32d6ec0939c7ff99059R5-R23) 

@anik120 did we not revert the changes? 
Were not the error faced here because before we did not set